content/docs/api-reference/params_show.md

```diff
@@ -0,0 +1,210 @@
+# dvc.api.params_show()
+
+Load <abbr>parameters</abbr> (name and values) tracked in a <abbr>DVC
+project</abbr>.
+
+```py
+def params_show(
+    *targets: str,  # Optional
+    repo: Optional[str] = None,
+    stages: Optional[Union[str, Iterable[str]]] = None,
+    rev: Optional[str] = None,
+    deps: bool = False,
```

Comment:
@dberenbaum @jorgeorpinel 

Should we drop `deps`? 

It doesn't feel that useful after adding support for `stages`

Reply:
The only use case I see is to combine param file `targets` and `deps=True` to discover which params in a param file are used by the DVC project. If that's an intended feature we should probably add an example (or change the last one).

Reply:
> If that's an intended feature we should probably add an example (or change the last one).

I believe it is the original idea behind `deps` in the CLI. 

Note that you don't necessarily need to combine it with `targets`, it could be also useful to discover which params from *any* param file are used by the DVC project

Reply:
If we don't see much use in retrospect, no need for an example. Yet OK to keep to reflect the CLI, IMO. Open an issue in core to deprecate `deps` from both CLI and API maybe?